jinja2/ext.py

```diff
@@ -434,6 +446,57 @@ class AutoEscapeExtension(Extension):
     pass
 
 
+class DebugExtension(Extension):
+    """
+    A {% debug %} tag that dumps the available variables, filters and tests.
+    This is roughly equivalent to the Django Template Language {% debug %} tag,
+    and typical usage like this:
+
+        <pre>{% debug %}</pre>
```

Comment:
Code blocks must be preceded with two colons: `::` or (preferably with non-Python) `.. codeblock:: html+jinja`.

Reply:
I had not appreciated the code was being processed for markup. I will look into this.

Reply:
Pygments recognizes `html+jinja`, use that over `html`.